ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_linalg.py

```diff
@@ -834,3 +834,52 @@ def test_jax_numpy_cond(
         x=x[0],
         p=p,
     )
+
+
+@st.composite
+def _get_dtype_inputs_for_multi_dot(draw):
```

Comment:
I think it's best to give a generic name to this function and move it to `array_helpers.py`.